src/main/scala/is/hail/annotations/ScalaToRegionValue.scala

```diff
@@ -0,0 +1,96 @@
+package is.hail.annotations
+
+import is.hail.expr._
+
+object ScalaToRegionValue {
```

Comment:
This stuff is a big convenience. Do we want to move it to the test package so nobody accidentally uses it for production code?

I'll leave that up to you.